assets/texts/fr/amoDescription.html

```diff
@@ -32,33 +32,42 @@
-	<li>Utilisation du raccourcis-clavier (par défaut <code>Ctrl</code>+<code>Maj</code>+<code>:</code>) pour accèder au sélecteur.</li>
-	<li>Intégration dans Firefox Sync, pour avoir les même paramètrages sur vos appareils.</li>
-	<li>Ajustement de l'apparence et de nombreux autres paramètrages selon vos goûts.</li>
+	<li>Utilisation du raccourcis-clavier (par défaut <code>Ctrl</code>+<code>Maj</code>+<code>:</code>) pour accéder au sélecteur.</li>
+	<li>Intégration dans Firefox Sync, pour avoir les même paramétrages sur vos appareils.</li>
+	<li>Ajustement de l'apparence et de nombreux autres paramétrages selon vos goûts.</li>
 	<li>La taille du sélecteur et des émojis est également ajustable.</li>
-	<li>Utilisation d'un icône coloré ou monochrome pour la barre d'outils.</li>
+	<li>Utilisation d'un icône colorée ou monochrome pour la barre d'outils.</li>
 	<li>Suivi du <a href="https://design.firefox.com/photon">Design Firefox Photon </a>.</li>
 	<li>Respect de la vie privée, avec aucun envoi de donnée.</li>
 	<li>Compatible avec Firefox sur Android</li>
-	<li>Déjà traduit dans plusieurs langues. <a href="https://github.com/rugk/awesome-emoji-picker/blob/master/CONTRIBUTING.md#Translations">Vous pouvez traduire la votre !</a></li>
+	<li>Déjà traduit dans plusieurs langues. <a href="https://github.com/rugk/awesome-emoji-picker/blob/master/CONTRIBUTING.md#Translations">Vous pouvez traduire la vôtre !</a></li>
 	<li>Les paramètres peuvent être gérés par votre administrateur système.</li>
 </ul>
 
 
-<b>📝 Dévelopement 📝</b>
-Ce module complémentaire est un logiciel gratuit, libre et "open-source" et est dévelopé sur GitHub. <a href="https://github.com/rugk/awesome-emoji-picker">Forker le sur GitHub</a> et contribuez.
-<a href="https://github.com/rugk/awesome-emoji-picker/contribute">Voici des tâches faciles à gèrer pour commencer.</a>
+<b>📝 Développement 📝</b>
+Ce module complémentaire est un logiciel gratuit, libre et "open-source" et est développé sur GitHub. <a href="https://github.com/rugk/awesome-emoji-picker">Forker le sur GitHub</a> et contribuez.
+<a href="https://github.com/rugk/awesome-emoji-picker/contribute">Voici des tâches faciles à gérer pour commencer.</a>
 
 Le développement de ce module a aussi été provoqué, car le développement d'un autre module nommé <a href="https://github.com/johannhof/emoji-helper/issues/127">“Emoji Helper” ou “Emoji Cheatsheet” est en suspend</a>.
 
 
+<b>🐬 Contribuer 🐬</b>
+<p>Vous pouvez facilement participer à ce projet et aider. Voici quelques idées :</p>
+<ul>
+	<li>📃 <a href="https://github.com/rugk/awesome-emoji-picker/blob/master/CONTRIBUTING.md#translations">Traduisez ce module en plusieurs langues !</a></li>
+	<li>🐛 <a href="https://github.com/rugk/awesome-emoji-picker/blob/master/CONTRIBUTING.md#coding">Réglez des problèmes simples et débutez dans le développement du module</a> (ou essayez une version de développement)</li>
```

Comment:
I think when I worked on the QR-Code generator I chose "extension" instead of "module". That's how my browser calls it.